### DISCUSSION TITLE
Which tools do you use to edit SVG icons?

### DISCUSSION THREAD

**Discussion by Snailedlt** [devicons/devicon#1787] [Polls]
By editing I mean things such as:
- Resizing
- Centering
- Combining paths
- Optimizing the SVG
- Creating plain/line/wordmark versions

**Comment by Snailedlt** [comment]
Seems like you can't vote for multiple options... So if you use multiple of the ones above, select the one you use the most and comment on the others :)

**Comment by Panquesito7** [comment]
- `iloveimg` (already voted).
- SVGO.
- `svgviewer.dev`.
- VSCode.
- PowerPoint for SVG editing.

**Comment by Snailedlt** [comment]
- Iloveimg (voted) 
- Inkscape
- SVGO (Specifically SVGOMG)
- vscode

**Comment by Gautam25Raj** [comment]
- Inkscape (Voted)
- SVGO.
- svgviewer.dev.
- VSCode.

**Comment by okkdev** [comment]
- Figma
- GodSVG
- Inkscape

**Reply by Snailedlt** [reply]
Interesting! Do you use Figma for editing SVG icons? If so, which parts can and can't Figma do. e.g. scaling, positioning, optimization, vectorize raster images, create plain/line version.

Also never heard of GodSVG before, I'll need to check that out!

**Reply by Snailedlt** [reply]
GodSVG looks great! I hope they make a vscode extension down the line. Would be really neat to have a good and lightweight SVG editor built into my code editor!

**Reply by okkdev** [reply]
Figma is not specifically made for it, but it's pretty decent at working with vector graphics. It has all the basic tools to modify vector graphics. No vectorizing or optimizing afaik (tho there might by plugins)

I love GodSVG for debugging SVGs. Idk how realistic a VSCode extension is as GodSVG itself is made using the Godot Game Engine, unless it's completely embedded or rewritten.
